.github/workflows/client.yml

```diff
@@ -1,6 +1,8 @@
-name: Go
+name: Go / keep-core
```

Comment:
I wonder if we really need to change the name. I know it would make it easier to differentiate the workflows on the meercode dropdown list, but in the filters we apply there we'll want to display all or no `Go` workflows, so differentiating is not that important, I think.
The downside of having the name change is that having some of the workflows with the ` / keep-core` name and some without them in the GH Actions view would look a bit inconsistent. Apart from that we would see the `/ keep-core` duplicated on the PR checks lists.
Both downsides are not that serious, so I'm ok with the change if it is really necessary.

Reply:
Yes, it was added to make it easier to navigate in Meercode. Okay, we can remove it and we'll see if that would be a problem navigating in Meercode. We can always modify the name if needed. 15d0c90